docs/howto/inspect_suite_database.ipynb

```diff
@@ -0,0 +1,473 @@
+       "      <th>3</th>\n",
+       "      <td>19831213T0700Z</td>\n",
+       "      <td>Quokka</td>\n",
+       "      <td>vld398.cmpd1.metoffice.gov.uk</td>\n",
+       "      <td>background</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>4</th>\n",
+       "      <td>19831213T0700Z</td>\n",
+       "      <td>Numbat</td>\n",
+       "      <td>vld398.cmpd1.metoffice.gov.uk</td>\n",
+       "      <td>background</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>5</th>\n",
+       "      <td>19831213T0700Z</td>\n",
+       "      <td>Wambenger</td>\n",
+       "      <td>vld398.cmpd1.metoffice.gov.uk</td>\n",
+       "      <td>background</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>6</th>\n",
+       "      <td>19831213T0800Z</td>\n",
+       "      <td>Quokka</td>\n",
+       "      <td>vld398.cmpd1.metoffice.gov.uk</td>\n",
+       "      <td>background</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>7</th>\n",
+       "      <td>19831213T0800Z</td>\n",
+       "      <td>Numbat</td>\n",
+       "      <td>vld398.cmpd1.metoffice.gov.uk</td>\n",
+       "      <td>background</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>8</th>\n",
+       "      <td>19831213T0800Z</td>\n",
+       "      <td>Wambenger</td>\n",
+       "      <td>vld398.cmpd1.metoffice.gov.uk</td>\n",
+       "      <td>background</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>9</th>\n",
+       "      <td>19831213T0900Z</td>\n",
+       "      <td>Quokka</td>\n",
+       "      <td>vld398.cmpd1.metoffice.gov.uk</td>\n",
+       "      <td>background</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "            cycle       name                   user_at_host batch_sys_name\n",
+       "0  19831213T0600Z     Quokka  vld398.cmpd1.metoffice.gov.uk     background\n",
+       "1  19831213T0600Z     Numbat  vld398.cmpd1.metoffice.gov.uk     background\n",
+       "2  19831213T0600Z  Wambenger  vld398.cmpd1.metoffice.gov.uk     background\n",
+       "3  19831213T0700Z     Quokka  vld398.cmpd1.metoffice.gov.uk     background\n",
+       "4  19831213T0700Z     Numbat  vld398.cmpd1.metoffice.gov.uk     background\n",
+       "5  19831213T0700Z  Wambenger  vld398.cmpd1.metoffice.gov.uk     background\n",
+       "6  19831213T0800Z     Quokka  vld398.cmpd1.metoffice.gov.uk     background\n",
+       "7  19831213T0800Z     Numbat  vld398.cmpd1.metoffice.gov.uk     background\n",
+       "8  19831213T0800Z  Wambenger  vld398.cmpd1.metoffice.gov.uk     background\n",
+       "9  19831213T0900Z     Quokka  vld398.cmpd1.metoffice.gov.uk     background"
+      ]
+     },
+     "execution_count": 103,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "from io import StringIO\n",
+    "pd.read_csv(\n",
```

Comment:
```python
NameError                                 Traceback (most recent call last)
<ipython-input-8-513363772c68> in <module>
      1 from io import StringIO
----> 2 pd.read_csv(
      3     StringIO(M.stdout),
      4     sep="|",
      5     index_col=False,

NameError: name 'pd' is not defined
```

Reply:
Yep, there needs to be a pandas import somewhere. I'm guessing that it worked for me because I'd loaded pandas in that session and it remained available, even when I deleted the code which had imported it. Jupyter Notebooks can be a pain on that front.